doc/index.rst

```diff
@@ -0,0 +1,25 @@
+.. gem-plotting-tools documentation master file, created by
+   sphinx-quickstart on Wed May 23 14:52:11 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+gem-plotting-tools package
```

Comment:
`gem-plotting-tools` should bechanged to `gemplotting`, reflecting the actual name of the `python` package